test/e2e/20_commit_signing.bats

```diff
@@ -6,31 +6,32 @@ load lib/gpg
 load lib/install
 load lib/poll
 
-tmp_gnupghome=""
-git_port_forward_pid=""
-
 function setup() {
   kubectl create namespace "${FLUX_NAMESPACE}" &> /dev/null
 
   # Install the git server, allowing external access
-  install_git_srv flux-git-deploy git_srv_result
-  # shellcheck disable=SC2154
-  git_ssh_cmd="${git_srv_result[0]}"
-  export GIT_SSH_COMMAND="$git_ssh_cmd"
+  install_git_srv git_srv_result
   # shellcheck disable=SC2154
-  git_port_forward_pid="${git_srv_result[1]}"
+  export GIT_SSH_COMMAND="${git_srv_result[0]}"
   # Teardown the created port-forward to gitsrv.
-  defer kill "$git_port_forward_pid"
+  defer kill "${git_srv_result[1]}"
```

Comment:
I _think_ this should be `defer "kill '${git_srv_result[1]}'"`?

Reply:
It's a number, so you don't really need to single-quote it. You can pass multiple arguments to defer since it expands them.

Reply:
https://github.com/fluxcd/flux/pull/2587/files#diff-806c4694241259b6acd056c83308cb4fR40, should be like this then :-).

Reply:
Yes, I missed that